.github/workflows/codeql.yml

```diff
@@ -0,0 +1,34 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - master
+      - "!renovate/**"
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches:
+      - master
+  schedule:
+    - cron: "0 0 * * 0"
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      # Initializes the CodeQL tools for scanning.
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v1
+        with:
+          languages: "javascript"
```

Comment:
Do they do anything separate for TypeScript?  Or does JavaScript cover all of it?

Reply:
There's no explicit TypeScript language AFAICT. We can probably skip the language but it's what I use in my JS projects.